compiler/rustc_trait_selection/Cargo.toml

```diff
@@ -19,6 +19,7 @@ rustc_infer = { path = "../rustc_infer" }
 rustc_lint_defs = { path = "../rustc_lint_defs" }
 rustc_macros = { path = "../rustc_macros" }
 rustc_query_system = { path = "../rustc_query_system" }
+rustc_serialize = { path = "../rustc_serialize" }
```

Comment:
Not blocking, but but what's this crate dependency for?

Reply:
for `rustc_index::newtype_index!`